composer.json

```diff
@@ -1,5 +1,8 @@
 {
     "name": "atoum/standard-edition",
+    "type": "package",
```

Comment:
"Only use a custom type if you need custom logic during installation" : there is no custom logic, maybe we should remove this attribute.